### PR TITLE
adds vf-nearest-location to the vf-wp-intranet tabs

### PR DESCRIPTION
**Pull request by kasprzyk-sz**

### CODE REVIEW

File: vf-components/vf-componenet-rollup/scripts.js [hunk 1]
```diff
@@ -19,6 +19,8 @@ vfGaIndicateLoaded();
 import { vfTabs } from 'vf-tabs/vf-tabs';
 vfTabs();
 
+import { vfLocationNearest } from 'vf-location-nearest/vf-location-nearest';
+
```
Comment (kasprzyk-sz) [inline vf-components/vf-componenet-rollup/scripts.js:23]: forgot about `vfLocationNearest();`

File: wp-content/themes/vf-wp/assets/scripts/scripts.js [hunk 2]
```diff
@@ -2055,6 +2055,44 @@ function emblBreadcrumbs() {
 })([Element.prototype, Document.prototype, DocumentFragment.prototype]); // Run it on default
 // emblBreadcrumbs();
 
+
+// Configure an object of your locations to detect
```
Comment (khawkins98) [inline wp-content/themes/vf-wp/assets/scripts/scripts.js:2059]: One thing: This JS (and `vfLocationNearest();`) should only run on the Intranet child theme. 

The easiest way to add might be to just insert it directly into the PHP template between some `<script></script>` tags.

Comment (kasprzyk-sz) [reply wp-content/themes/vf-wp/assets/scripts/scripts.js:2059]: Ok, back to the drawing board!

Comment (kasprzyk-sz) [reply wp-content/themes/vf-wp/assets/scripts/scripts.js]: Updated

**Comment by khawkins98** [comment]
Looks good.

**Comment by khawkins98** [comment]
Nice work. Look forward to seeing if it works in the wild 😊

**Comment by kasprzyk-sz** [comment]
Let's hope for the best :grin:

**Pull request merged**